README.md

```diff
@@ -214,8 +214,8 @@ stopped before switching. If the specified version is already in use, the `use`
 command won't stop it, but will initialize its data directory and starts it if
 it's not already running.
 
-```
-pgenv use 10.4
+``` sh
```

Comment:
I would not include the `sh` here because the block has output lines that are not shell scripting, just text.

Reply:
Sounds like literate programming, not "makrdown", but agree. Fix in beb5f98